packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php

```diff
@@ -395,7 +397,7 @@
         ->assertJsonPath('cart.shipping_address.cart_id', $cartId);
 });
 
-it('should store the orders for guest user', function () {
+it('should store the orders of simple product for guest user', function () {
```

Comment:
This test case is of listing so rename this to "should display the cart items from the cart" and add remaning 6 testcases for add item to cart.